Quaver.Shared/Database/Maps/OtherGameMapDatabaseCache.cs

```diff
@@ -199,7 +199,7 @@ private static List<OtherGameMap> LoadOsuBeatmapDatabase()
                     var newMap = new OtherGameMap()
                     {
                         Md5Checksum = map.BeatmapChecksum,
-                        Directory = map.FolderName.Trim(),
+                        Directory = map.FolderName,
```

Comment:
You merged the wrong commit because this isn't supposed to disappear

Reply:
wtf